clang/test/CheckedCRewriter/inline_anon_structs.c

```diff
@@ -74,7 +74,7 @@ struct alpha {
 };
 struct alpha *al[4];
 	//CHECK_NOALL: struct alpha *al[4];
-	//CHECK_ALL: _Ptr<struct alpha> al _Checked[4];
+	//CHECK_ALL: _Ptr<struct alpha> al _Checked[4] = {((void *)0)};
```

Comment:
Why only one initializer? Does this make sense? Seems like Checked C's `clang` should reject this?

Reply:
It fills in the rest of the array with `(void*)0`. You just need to give an initializer.